app/src/main/java/de/tap/easy_xkcd/fragments/NestedPreferenceFragment.java

```diff
@@ -511,8 +511,6 @@ public void run() {
 
             case ADVANCED:
                 addPreferencesFromResource(R.xml.pref_advanced);
-                findPreference(MOBILE_ENABLED).setEnabled(MainActivity.fullOffline | MainActivity.fullOfflineWhatIf);
```

Comment:
Shouldn't you move this to somewhere else instead of just deleting it?

Reply:
I added `android:dependency` to the preference in xml, but this allows only one other preference as dependency. I added this line back to the PR.